addons/mail/static/src/core/web/chatter.js

```diff
@@ -197,19 +200,23 @@ export class Chatter extends Component {
     /**
      * @returns {string}
      */
-    get toFollowersText() {
-        const threadName = this.state.thread.displayName || this.state.thread.name;
-        const toFollowersText = threadName
-            ? sprintf(
-                  _t(
-                      '<span class="fw-bold">To:</span> <span class="fst-italic">Followers of</span> <span class="fw-bold">"%(thread name)s"</span>.'
-                  ),
-                  { "thread name": escape(threadName) }
-              )
-            : _t(
-                  '<span class="fw-bold">To:</span> <span class="fst-italic">Followers of</span> this document.'
-              );
-        return markup(toFollowersText);
+    get toRecipientsText() {
+        const followers = this.state.thread.followers.slice(0, 5).map(({ partner }) => {
+            if (partner === this.store.self) {
+                return `<span class="text-muted" title="${escape(partner.email)}">me</span>`;
+            }
+            return `<span class="text-muted" title="${escape(partner.email)}">${escape(
+                partner.emailWithoutDomain
+            )}</span>`;
+        });
+        const formatter = new Intl.ListFormat(
+            this.store.env.services["user"].lang?.replace("_", "-"),
+            { type: "unit" }
+        );
+        if (this.state.thread.followers.length > 5) {
+            followers.push("…");
+        }
+        return markup(formatter.format(followers));
```

Comment:
User-customized content (email) is escaped, rest of string is static. Therefore markup is safe.
All this is done in JS due to separator `,` that needs to take l18n into account (e.g. `、` in Japanese).